stock_inventory_preparation_filter/models/stock_inventory.py

```diff
@@ -61,7 +57,7 @@ def _get_available_filters(self):
         return res_filters
 
     filter = fields.Selection(
-        selection=_get_available_filters, string='Selection Filter',
+        selection=_selection_filter, string='Selection Filter',
```

Comment:
Remove the redefinition of the field. You are even breaking inheritance, because you are redefining the selection method without quotes, which makes this non inheritable.